src/cmap/auth/gssapi.ts

```diff
@@ -12,10 +12,22 @@ import {
 import { Callback, ns } from '../../utils';
 import { AuthContext, AuthProvider } from './auth_provider';
 
+/** @public */
+export const CANONICALIZATION_VALUES = [
```

Comment:
I'm wondering if we could take the same approach here as we do elsewhere with enums, (e.g., `ReadPreferenceMode`) with an `export const ... Object.freeze` and `export type ... typeof ... keyof` for consistency, that way we don't have use actual strings in our code

Reply:
I've updated to use the same pattern.

```
export const CanonicalizationProperties = Object.freeze({
  on: true,
  off: false,
  none: 'none',
  forward: 'forward',
  forwardAndReverse: 'forwardAndReverse'
} as const);
```